src/windows/config.c

```diff
@@ -71,6 +71,8 @@ static inline void init_config_file() {
                    TEXT("false"), &config.redirect_output_log);
     load_path_file(config_path, TEXT("General"), TEXT("target_assembly"),
                    DEFAULT_TARGET_ASSEMBLY, &config.target_assembly);
+    load_path_file(config_path, TEXT("General"), TEXT("boot_config_override"),
+                   TEXT(""), &config.boot_config_override);
```

Comment:
Wouldn't assigning empty string by default break the if checks?
```cs
if (config.boot_config_override) {
```

Other paths are inited with a NULL instead.
```cs
    load_path_file(config_path, TEXT("Il2Cpp"), TEXT("coreclr_path"), NULL,
                   &config.clr_runtime_coreclr_path);
    load_path_file(config_path, TEXT("Il2Cpp"), TEXT("corlib_dir"), NULL,
                   &config.clr_corlib_dir);
```

Reply:
Not all paths were, the mono override one isn't for example.

But I tested on Windows just now anyway with logs: it never passes this check with a blank value.